.mvn/wrapper/maven-wrapper.properties

```diff
@@ -0,0 +1,18 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+# 
+#   http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.8.5/apache-maven-3.8.5-bin.zip
```

Comment:
we can't use 3.8.5, we're hit by https://issues.apache.org/jira/browse/MNG-7432

Reply:
Oh, I wasn't aware of the open issue when I commented on the PR last time. I guess we can safely use 3.8.1 as recommended in the contributing.md.

Reply:
3.8.4 is what's used on CI

Reply:
Of course this kind of regression is a prime argument for using the Maven wrapper!